### DISCUSSION TITLE
Inpaint/Oupaint

### DISCUSSION THREAD

**Discussion by riazuddinshaik** [lllyasviel/Fooocus#3274] [Q&A] [answered]
@mashb1t  Hi. I tried to generate the printing name and logo by uploading image in image prompt with stp at 0.86 and weght 0.96 using "mix image prompt and inpaint" feature. I later masked the another tshirt image under inpaint section. The text and logo generated was weird. Its UNREADABLE. How to generate atleast 70-80%% of graphics? Thankyou.

**Comment by mashb1t** [comment, marked as answer]
Not sure what you're trying to achieve, but https://huggingface.co/spaces/yisol/IDM-VTON might be more suited for clothing / try-on than image prompts in Fooocus as it also has pose support for the underlying skeleton of the subject.

**Reply by riazuddinshaik** [reply]
@mashb1t Thankyou so much for your reply and reference. Could you assist me what is the difference between fooocus inpaint and IDM inpaint method because IDM Vton also need to mask for inpainting. Is any adapter version or anything is making difference between both?

**Reply by mashb1t** [reply]
@riazuddinshaik it's also using stable diffusion, but is completely different in their methodology. You can read how they do it in the paper here: https://arxiv.org/abs/2403.05139

**Reply by riazuddinshaik** [reply]
I'm really surprised with kind of quick response. Thankyou for your support. I read and get back to you

**Comment by ILikeToasters** [comment]
Are you trying to add text and simple graphics to a shirt when you generate? If so this might help but not sure if that is your goal. https://youtu.be/MnGYEB_oOqE
